.gitignore

```diff
@@ -1,5 +1,10 @@
 .DS_Store
 .git-completion.bash
+
+# ignore data files
```

Comment:
please dont change the main .gitignore - there may be reasons to save some .csv or .txt files  in git.

you can put your own .gitignore with this into your own students dir, and then it will only apply to your files.